adobecom/milo
# Quiz entry block

1. Add Analytics for Quiz Entry Block
2. Integrate Auto-Complete for Open Text Field on UAR Exposed Front Door
3. remove "results-container" from previous feedback

Resolves: 
[MWPW-147031](https://jira.corp.adobe.com/browse/MWPW-147031)
[MWPW-146028](https://jira.corp.adobe.com/browse/MWPW-146028)
[MWPW-146080](https://jira.corp.adobe.com/browse/MWPW-146080)

**Test URLs:**
- Before: https://quiz-entry-block--milo--adobecom.hlx.page/drafts/colloyd/quiz-entry/?martech=off
- After: https://quiz-entry-block--milo--jackysun9.hlx.page/drafts/colloyd/quiz-entry/?martech=off

How to test:
1. Analytics
a) use this url: https://quiz-entry-block--milo--jackysun9.hlx.page/drafts/colloyd/quiz-entry/?martech=off&debug=quiz-entry
click different cards and check daa-ll value on continue button
type in different words and click continue button and check daa-ll value on continue button
b) use this url: https://quiz-entry-block--milo--jackysun9.hlx.page/drafts/colloyd/quiz-entry/?martech=off
use cards or mlfield to get through to result page and check pageloadHash in localStorage quiz key

2. Autocomplete
use this url: https://quiz-entry-block--milo--jackysun9.hlx.page/drafts/colloyd/quiz-entry/?martech=off
type any words in mlField, you will see autocomplete

Feature work: need address some css issues in another story
